### PR TITLE
Show the ProfileTotalCountView as soon as 'total_views_count' is available from the API

### PR DESCRIPTION
**Pull request by colinta**
Show/hide the `badgeButton`, too, either in the avatar view or total count view.

The behavior is a bit odd if the `total_views_count` is not always present on the user, since we don't load users if they've already been loaded.

### CODE REVIEW

File: Sources/Controllers/Profile/Views/ProfileTotalCountView.swift [hunk 1]
```diff
@@ -98,22 +89,29 @@ extension ProfileTotalCountView {
...
 
-        let count = ElloAttributedString.style(countText + " ", [
-            NSForegroundColorAttributeName: UIColor.blackColor(),
-            NSParagraphStyleAttributeName : paragraphStyle
-        ])
-        attributed.appendAttributedString(count)
-        attributed.appendAttributedString(totalViewsText)
-        return attributed
-    }
+        if let count = count where count != "" {
```
Comment (steam) [inline Sources/Controllers/Profile/Views/ProfileTotalCountView.swift]: We could use `count.isEmpty` here.

File: Sources/Model/User.swift [hunk 2]
```diff
@@ -52,6 +52,14 @@ public final class User: JSONAble {
     public var backgroundPosition: String?
     public var onboardingVersion: Int?
     public var totalViewsCount: Int?
+    public var formattedTotalCount: String? {
```
Comment (steam) [inline Sources/Model/User.swift]: This should probably be a function. :trollface:

**Comment by jayzes** [comment]
Does this require that we push that null attribute on the API response before it goes to the store?

**Comment by colinta** [comment]
@jayzes No, we don't need anything - it looks for an int, but defaults to `nil`.

**Comment by colinta** [comment]
@jayzes @Justin-Holmes btw we're looking for an int in the json called `total_views_count`

**Comment by jayzes** [comment]
@colinta FWIW, wonder if we should make it a `double` or a `long` (not sure which swift has) just to future-proof it

**Comment by colinta** [comment]
@jayzes The default `Int` type is pretty robust, it's not the C `int` type of yesteryear:

```
> Int.max
$R0: Int = 9223372036854775807
```

**Comment by jayzes** [comment]
@colinta excellent!

**Pull request merged**